packages/material-renderers/src/controls/MaterialTimeControl.tsx

```diff
@@ -75,16 +82,32 @@ export const MaterialTimeControl = (props: ControlProps) => {
     : null;
   const secondFormHelperText = showDescription && !isValid ? errors : null;
 
+  const updateChild = useCallback(() => setKey((key) => key + 1), [setKey]);
```

Comment:
technically `setKey` is not required to be in the dependency list as it's guaranteed to be stable by React, however adding it doesn't break anything.